lib/error_tracker/migration/sqlite/v02.ex

```diff
@@ -0,0 +1,44 @@
+defmodule ErrorTracker.Migration.SQLite.V02 do
+  @moduledoc false
+
+  use Ecto.Migration
+
+  def up do
+    create table(:error_tracker_meta, primary_key: [name: :key, type: :string]) do
+      add :value, :string
```

Comment:
Should this be nullable? It is required in the Postgres equivalent.


```suggestion
      add :value, :string, null: false
```

Reply:
Updated in https://github.com/elixir-error-tracker/error-tracker/pull/31/commits/6324adc9e0841ac30c24d8ad8fa5bbe5e96ca9d9